tests/test_examples.py

```diff
@@ -9,6 +9,9 @@ def classcase(name):
     return "".join(x.capitalize() for x in name.replace("-", "_").split("_"))
 
 
+@unittest.skip(
```

Comment:
This test is dependent on examples folder being here, so we are skipping until we discuss testing strategy.